docker_wrapper.py

```diff
@@ -4,10 +4,33 @@
 import os
 import sys
 
-mirror = "gcr.azk8s.cn"
-namespace = "google_containers"
-prefix = "gcr.io"
-specialPrefix = "k8s.gcr.io"
+#  xxx:yyy -> dockerhub.azk8s.cn/library/xxx:yyy
+#
+#  gcr.io/xxx/yyy:zzz -> gcr.azk8s.cn/xxx/yyy:zzz
+#			
+#  k8s.gcr.io/xxx:yyy => gcr.io/google-containers/xxx:yyy -> gcr.azk8s.cn/google-containers/xxx:yyy
+#		
+#	 quay.io/xxx/yyy:zzz -> quay.azk8s.cn/xxx/yyy:zzz
+#
+
+converts = [
+	{
+			'prefix': '',
+			'replace': lambda x : 'dockerhub.azk8s.cn/library/' + x,	
```

Comment:
user may have set docker mirror and do not want to use own Azure mirror, so we should pull image directly.
I think this is not necessary.